app/src/main/java/tech/pauly/findapet/data/models/Age.java

```diff
@@ -0,0 +1,8 @@
+package tech.pauly.findapet.data.models;
+
+public enum Age {
+    Baby,
```

Comment:
Added chore to come back to the case parsing issue (hopefully without writing deserializers 🤞) https://www.pivotaltracker.com/story/show/157179649